docs/DeveloperGuide.md

```diff
@@ -38,22 +38,67 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 ### Product scope
 
 **Target user profile**:
+* Freelance cheesemaker
+* Runs home-based business
+* Prefers desktop apps over other types
+* Can type fast
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**:
+1. Specific properties of each cheese
+2. Track order status of each cheese (either by batch or by individual)
 
 
 ### User stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | [user] | [requirement] | [purpose] |
+| Priority | As a …​                      | I want to …​                     | So that I can …​                                                        |
+| -------- | ----------------------------| ------------------------------- | ---------------------------------------------------------------------- |
+| `* * *`  | New user                    | Input a new order |  |
+| `* * *`  | New user                    | Remove an existing order |  |
+| `* * *`  | New user                    | Add cheese entries |  |
+| `* * *`  | New user                    | Mark a sample order as delivered |  |
+| `* * *`  | New user                    | Search for a customer | Find the customer’s contact information |
+| `* *`    | New user                    | Save the data input | Retrieve the same information later |
+| `* *`    | User with some familiarity  | View a summary of my inventory | See if there is a need to increase production |
+| `* *`    | User                        | See the introduction message |  |
+| `*`      | User with some familiarity  | Search up orders of a specific customer | Efficiently find the order status |
+
+[More to be added]
```

Comment:
Is this needed?

Reply:
Maybe they can serve as examples of low priority tasks?

Reply:
Should be fine I guess.